pkg/multitenant/mtinfopb/info.go

```diff
@@ -61,6 +61,7 @@ var TenantServiceModeValues = map[string]TenantServiceMode{
 	"none":     ServiceModeNone,
 	"external": ServiceModeExternal,
 	"shared":   ServiceModeShared,
+	"stopping": ServiceModeStopping,
```

Comment:
why does this require adding a new service mode? why not use `TenantDataState`?

Reply:
DataState describes the state of their persisted data, not their service. i.e. if you are half-way done restoring or importing or reverting them, the data is inconsistent vs the data is correct, but you can choose if and how to serve it -- but that actual choice is separate, and represented by svc mode.

Reply:
Also note that this "new service mode" isn't all that new: we started adding it back in back in https://github.com/cockroachdb/cockroach/pull/117677.

Reply:
Ah, thanks for the context.